### ISSUE TITLE
[1.18.2 Suggestion] Add blacklist for Item User

### ISSUE THREAD

**Issue by itsdinkd** [PrinceOfAmber/Cyclic#2124]
Minecraft Version:  1.18.2

So using the item user you can exploit a few mods. Please add in a ignore_list for this block into its config.

**Comment by Lothrazar** [comment]
good idea ill look into it. might be an item data tag instead of config list
